.travis.yml

```diff
@@ -15,6 +15,12 @@ matrix:
 notifications:
   email: false
 script:
-  - mix compile --warnings-as-errors
+  - >
```

Comment:
@yordis don't we want something like this for Guardian?

Reply:
That was my intent. I am waiting for you to respond about the Elixir version so I replicate the same setup with more features on it as I did on ueberauth